tests/main_flows.spec.ts

```diff
@@ -114,6 +126,17 @@ describe('main_flows_tests', function() {
       PROGRAM_ID,
     );
   });
+  
+  it('Update pool fee', async function() {
+    await SarosSwapService.updatePoolInfo(
```

Comment:
should not test like this, this scenario requires create a new pool from scratch then update and assert the different